<launch>
    <node name="data_collection" pkg="jackal_bsc" type="data_collection.py" output="screen"/>
</launch>